### ISSUE TITLE
Is there a way to view the console log as a text file?

### ISSUE THREAD

**Issue by Crooncode** [kolton/d2bot-with-kolbot#1414]
I'd like to be able to track game times and experience outside of the manager console screen. Does the console create a consolelog.txt or something similar?

**Comment by DarkHorseDre** [comment]
d2bot-with-kolbot-master/logs has a file called console.rtf. I think its enabled by default?

Config.LogExperience = false; // Print experience statistics in the manager. - character config

**Comment by Crooncode** [comment]
Ah thanks - that looks like exactly what I'm after - but even with that logexperience set to true it is only showing in the manager and not saving into that console.rtf file. any ideas?

**Comment by firekil** [comment]
Go to the folder d2bs and click on d2bs.log. That's everything that gets printed in the console.

**Comment by DarkHorseDre** [comment]
yeah i tried to look for how to control that file and i cant find a reference.. (d2bs.log is enabled via d2bs.ini but that file is bloated, but use that if we cant get console.rtf to populate (much cleaner file))

**Comment by Crooncode** [comment]
Thanks - yeh I can't seem to find the control for the console.rtf. please shout out if you do figure it out

**Comment by 5noop** [comment]
You can export the content of the manager console to console.rtf with a right click -> export. Not sure if it can be done automatically.

**Comment by DarkHorseDre** [comment]
ahhhhhh!!!!
